Cargo.toml

```diff
@@ -15,24 +15,24 @@ members = ["sqlxmq_macros", "sqlxmq_stress"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-sqlx = { version = "0.7.1", features = ["postgres", "chrono", "uuid"] }
-tokio = { version = "1.8.3", features = ["full"] }
-dotenv = "0.15.0"
-chrono = "0.4.19"
-uuid = { version = "1.1.2", features = ["v4"] }
-log = "0.4.14"
-serde_json = "1.0.64"
-serde = "1.0.124"
+sqlx = { version = "~0.8.1", features = ["postgres", "chrono", "uuid"] }
```

Comment:
Could we just have `version = ">=0.7, <1.0"`?
There is no reason to pin it to a very specific version except if it's known that it would break

Reply:
This PR should also bump the semver version of this crate to `0.6.0` since sqlx is a public dependency.